server/game/gamesteps/plotphase.js

```diff
@@ -51,27 +49,15 @@ class PlotPhase extends Phase {
         });
     }
 
-    removeActivePlots() {
-        for(const player of this.game.getPlayers()) {
-            player.removeActivePlot();
-        }
-    }
-
-    flipPlotsFaceup() {
-        for(const player of this.game.getPlayers()) {
-            player.flipPlotFaceup();
-        }
-    }
-
     recyclePlots() {
         for(const player of this.game.getPlayers()) {
             player.recyclePlots();
         }
     }
 
     getActivePlots() {
```

Comment:
maybe rename to `getSelectedPlots()` as the plots are not active yet?

Reply:
Yup. I want to totally change how plot selection is done so this method will go away eventually, but for now, the name will be updated.